Crawler/page.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+import re
+import sys
+import requests
+from bs4 import BeautifulSoup
+import urllib2
+import re
+import csv
+import unicodedata as ud
+import datetime
+import os.path
+import shutil
+import datetime
+
+
+# def teste():
+
+	# link = 'https://seekingalpha.com/article/4091321-google-home-competitive-strengths-fast-growing-smart-speaker-market'
+
+	
+	
+def teste(link,brand):
+	#brand = 'TESTE'
+
+	opener = urllib2.build_opener()
+	opener.addheaders = [('User-agent', 'Mozilla/5.0')]
+	response = opener.open(link)
+	
+	file = open('acao1.csv', 'a')
+	
+	
+	soup = BeautifulSoup(response, 'lxml')
+	
+	#get Title
+	seila5 = soup.find('h1',{'class':'has-title-test'})
```

Comment:
![giphy](https://user-images.githubusercontent.com/16463855/28851984-6d578de2-76fc-11e7-8ac4-2dceb2495c7f.gif)
Could you please change those variable names?